testing-standard.md

```diff
@@ -105,3 +210,38 @@ someModule._action = function(number) {
   return number;
 };
 ```
+
+### Testing the result of callbacks
+Mocha makes this easy out of the box, we can pass a `done` callback to each `it` block if we're running asynchronous code in our test. For example:
+
+```javascript
+  it('the function calls back with the number 1337', function(done) {
+    myTestFunctionCall(function(result) {
+      expect(result).toEqual(1337);
+      done();
+    });
+  ); 
+```
+
+### Testing the result of promises
+Using the [chai-as-promised](https://github.com/domenic/chai-as-promised/) library makes this very easy:
+
+```javascript
+  it('the function calls back with the number 1337', function() {
+    expect(myTestFunctionCall()).to.eventually.equal(1337);
+  ); 
+```
+
+# 2.0 Selenuim Tests
+
+TODO
+
+# 3.0 System Tests
+
+## Each test should:
+* Use product codes that are guaranteed to be available
+* Use dynamic dates that are a fixed amount of time in the future
+* Not assert against any dynamic data (eg. content) that may change unexpectedly
```

Comment:
You're dancing around the topic a bit here, but we should specifically call out the need to fixture data and that the unit test suite should be capable of being ran without any backing services.